CONTRIBUTING.md

```diff
@@ -184,3 +186,33 @@ See other exercises, e.g. [acronym](https://github.com/exercism/java/tree/master
 
 Hopefully that should be enough information to help you port an exercise to the Java track.
 Feel free to open an issue or post in the [Gitter exercism/java room](https://gitter.im/exercism/java) if you have any questions and we'll try and answer as soon as we can.
+
+## Updating the READMEs
+
+The `README.md` files are generated from the exercise descriptions in [problem specifications](https://github.com/exercism/problem-specifications/tree/master/exercises).
+They need to be regenerated regularly so that any changes to the descriptions in problem specifications propagate to our REAMEs.
+This can be done using [configlet](https://github.com/exercism/configlet/releases):
+
+  1. Download configlet and put it somewhere in your PATH
```

Comment:
`PATH` to `$PATH`?

Reply:
Hm, I don't really mind which one we use. `PATH` is what we have in the adding a new exercise section, but I'm happy to change both if you think that's best 🙂

Or we could keep it as `PATH` but make it a link to https://en.wikipedia.org/wiki/PATH_(variable) to make it clear exactly what we mean? 🙂

Reply:
Either will be fine - it was a minor nitpick :slightly_smiling_face:

Reply:
Okay, will add the link! 🙂